recipes-domd/domd-image-weston/domd-image-weston.bbappend

```diff
@@ -2,10 +2,17 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 FILESEXTRAPATHS_prepend := "${THISDIR}/../../inc:"
 FILESEXTRAPATHS_prepend := "${THISDIR}/../../recipes-domx:"
 
-XT_PRODUCT_NAME ?= "prod_gen3_test"
+XT_PRODUCT_NAME ?= "prod-gen3-test"
+XT_MANIFEST_FOLDER ?= "prod_gen3_test"
```

Comment:
why do you need this variable? as per my understanding it will be set automatically by the code below

Reply:
You are right, removed